pom.xml

```diff
@@ -46,8 +47,23 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <version.jackson>2.7.4</version.jackson>
+        <version.spring>4.1.6.RELEASE</version.spring>
+        <version.slf4j>1.7.13</version.slf4j>
+        <version.log4j>2.4.1</version.log4j>
     </properties>
 
+    <modules>
+        <module>fahrschein</module>
```

Comment:
I prefer keeping the name of the main artifact the same as in previous versions, it should make updating less confusing.